lib/payload.json

```diff
@@ -127,8 +131,12 @@
         "wind": {
           "type": "object",
           "properties": {
-            "speed": {
-              "description": "Wind speed (m/s)",
+            "avgSpeed": {
+              "description": "Average wind speed (m/s)",
+              "$ref": "#/definitions/speed"
+            },
+            "gustSpeed": {
+              "description": "Gust wind speed (m/s)",
               "$ref": "#/definitions/speed"
             },
             "direction": {
```

Comment:
I think this is a good change. I think it is indeed useful domain knowledge to differentiate between average and gust wind speeds.

Can you turn it into `averageSpeed` though?

Reply:
OK, this is done.